### PR TITLE
Security council mgmt  deployment factories

### PR DESCRIPTION
**Pull request by DZGoldman**
deploys all but election-gov contracts (see deployment.md for steps)

### CODE REVIEW

**Review by godzillaba — COMMENTED** [review]
looks good. I haven't looked at the tests yet

File: src/security-council-mgmt/factories/AddressAliasHelper.sol [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+library AddressAliasHelper {
+    uint160 constant offset = uint160(0x1111000000000000000000000000000000001111);
+
+    /// @notice Utility function that converts the address in the L1 that submitted a tx to
+    /// the inbox to the msg.sender viewed in the L2
+    /// @param l1Address the address in the L1 that triggered the tx to L2
+    /// @return l2Address L2 address as viewed in msg.sender
+    function applyL1ToL2Alias(address l1Address) internal pure returns (address l2Address) {
+        l2Address = address(uint160(l1Address) + offset);
```
[the remark this answers is not in the document]
Comment (DZGoldman) [reply src/security-council-mgmt/factories/AddressAliasHelper.sol:12]: this was a lazy copy-paste b/c the import was complaining about the compiler version

[the remark this answers is not in the document]
Comment (godzillaba) [reply src/security-council-mgmt/factories/AddressAliasHelper.sol:12]: ahh ok gotcha

[the remark this answers is not in the document]
Comment (gzeoneth) [reply src/security-council-mgmt/factories/AddressAliasHelper.sol:12]: where did you copy that? the version in nitro-contracts should have unchecked
https://github.com/OffchainLabs/nitro-contracts/blob/main/src/libraries/AddressAliasHelper.sol
and it have `pragma solidity ^0.8.0;` which should be ok to import

**Comment by godzillaba** [comment]
should we do `undoL1ToL2Alias` on `_l2SecurityCouncilManager` on this line? or will we just pass in the unaliased address as the parameter? 

https://github.com/ArbitrumFoundation/governance/blob/e84ab1f134da12f13a29a60ee7c4baee5d9418ef/src/security-council-mgmt/factories/L1SecurityCouncilMgmtFactory.sol#L100

File: src/security-council-mgmt/factories/AddressAliasHelper.sol (hunk 1 above)
[the remark this answers is not in the document]
Comment (DZGoldman) [reply src/security-council-mgmt/factories/AddressAliasHelper.sol]: yeah idk. updated

**Comment by DZGoldman** [comment]
closing for now in light of refactor. will either reopen deployment factories stuff on this same branch (when ready) or on a fresh branch

**Pull request closed without merging**